packages/hash/frontend/src/pages/signup.page/account-setup-form.tsx

```diff
@@ -123,40 +158,78 @@ export const AccountSetupForm: FunctionComponent<AccountSetupFormProps> = ({
                       }}
                       onBlur={field.onBlur}
                       autoFocus
-                      className={tw`w-64 border-1 ${
-                        shortnameError
-                          ? "border-red-300 focus:border-red-500"
-                          : "border-gray-300 focus:border-blue-500"
-                      } border-solid focus:outline-none rounded-lg h-11 py-6 pl-9 pr-5 mr-7`}
+                      style={{
+                        borderColor: shortnameError
+                          ? "#FCA5A5" // focus:border-red-500
+                          : "#D1D5DB", // focus:border-blue-500"
+                        borderRadius: "0.5rem",
+                        borderStyle: "solid",
+                        borderWidth: 1,
+                        height: "2.75rem",
+                        marginRight: "1.75rem",
+                        paddingBottom: "1.5rem",
+                        paddingLeft: "2.25rem",
+                        paddingRight: "1.25rem",
+                        paddingTop: "1.5rem",
+                        width: "16rem",
+                        // focus:outline-none
+                      }}
                       placeholder="example"
                       autoComplete="off"
                     />
                   )}
                 />
 
                 <span
-                  className={tw`absolute text-gray-400 left-5 top-1/2 -translate-y-1/2`}
+                  style={{
+                    position: "absolute",
+                    left: "1.25rem",
+                    top: "50%",
+                    transform: "translateY(-50%)",
+                    color: "#9CA3AF",
+                  }}
                 >
                   @
                 </span>
               </div>
               <div
-                style={{ minHeight: 50 }}
-                className={tw`transition-opacity max-w-sm flex items-center border-1 ${
-                  shortnameError ? "border-red-300" : "border-blue-300"
-                } rounded-md px-3.5`}
+                style={{
+                  alignItems: "center",
+                  borderColor: shortnameError ? "#FCA5A5" : "#D1D5DB",
+                  borderRadius: "0.375rem",
+                  borderWidth: 1,
+                  display: "flex",
+                  maxWidth: "24rem",
+                  minHeight: 50,
```

Comment:
does this need a unit? I forget if JSX will append px or not

Reply:
Yep, numbers are converted to pixels by React